src/main/java/appeng/services/export/MinecraftItemCSVExporter.java

```diff
@@ -243,7 +244,7 @@ public String apply( @Nullable final Item input )
 				}
 				catch( final Exception ignored )
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)